.stylelintrc.json

```diff
@@ -0,0 +1,85 @@
+{
+  "rules": {
```

Comment:
Is there an existing config that we could extend?

Reply:
This is the config that I came up with. There are some existing configs like:
https://github.com/stylelint/stylelint-config-standard
https://github.com/stylelint/stylelint-config-recommended
https://github.com/JetBrains/stylelint-config
And lots more, so it's up to you guys to decide if we want to create our own, extend the default or pick someone elses.
I have just found a nice generator for configs https://maximgatilin.github.io/stylelint-config/